packages/ui/src/working-groups/hooks/useOpenings.ts

```diff
@@ -43,7 +43,7 @@ export const useOpenings = ({ groupId: group_eq, type, openingsPositionType }: U
     [loading, data]
   )
 
-  const sortedOpenings= openings.slice().sort((a, b) => a.workingGroupOpening.updatedAt - b.workingGroupOpening.updatedAt)
+  const sortedOpenings= openings.slice().sort((a, b) => a.leadId - b.leadId)
```

Comment:
please describe this change. looks like this would sort all openings by the `ID` of the lead membership vs chronologically.